tools/wpt/runner.ts

```diff
@@ -154,7 +154,6 @@ export async function runSingleTest(
         harnessStatus = JSON.parse(line.slice(5));
       } else {
         stderr += line + "\n";
-        console.error(line);
```

Comment:
This was confusing test results by printing out-of-order

Reply:
Good catch 👍